config/local/cloud-gateway-service.yml

```diff
@@ -2,6 +2,7 @@ apiml:
     cloudGateway.registry.enabled: true
     security.x509.registry.allowedUsers: Zowe Service
     service:
+        forwardClientCertEnabled: true
```

Comment:
would it make sense to put this new property under `security.x509` rather than `service`?

Reply:
This is not the new parameter. I just made it enabled for the local development.

Reply:
Ok I see, thanks for the clarification.